build.sbt

```diff
@@ -171,6 +172,21 @@ lazy val akkaHttp = project
   )
   .dependsOn(coreJVM)
 
+lazy val finch = project
+  .in(file("finch"))
+  .settings(name := "caliban-finch")
+  .settings(commonSettings)
+  .settings(
+    crossScalaVersions := Seq("2.12.10"),
+    libraryDependencies ++= Seq(
+      "com.github.finagle" %% "finchx-core"      % "0.31.0",
+      "com.github.finagle" %% "finchx-circe"     % "0.31.0",
+      "dev.zio"            %% "zio-interop-cats" % "2.0.0.0-RC10",
+      "org.typelevel"      %% "cats-effect"      % "2.0.0"
```

Comment:
Btw you can use 2.1.1 for cats-effect

Reply:
my bad I think this was something I changed that must not have survived merge conflict resolution with master